src/main.cpp

```diff
@@ -632,6 +633,23 @@ void print_help(string command) {
 		"Example: bspguy unembed c1a0.bsp\n"
 	);
 	}
+	else if (command == "exportobj") {
+	logf(
+		"exportobj - Export bsp geometry to obj [WIP].\n\n"
+
+		"Usage:   bspguy exportobj <mapname>\n"
+		"Example: bspguy exportobj c1a0.bsp\n"
+	);
+	}
+	else if (command == "editor" || command == "empty") {
+	logf(
+		"editor -\n"
+		"empty - Open empty bspguy window.\n\n"
+
+		"Usage:   bspguy editor\n"
+		"Usage:   bspguy empty\n"
+	);
+	}

```

Comment:
The tool isn't meant for creating new maps. I don't think this is needed for merging either.

Reply:
For example I don't like to open map from console, I not a linux user)))

Reply:
I have it set up as my default program, so double-clicking a map opens the editor. You can also drag and drop a map onto the executable to open the editor.

Reply:
The problem is this program is both a command-line tool and a windows app. If no arguments are given, should you see the 3D view or a help menu? Depends on what you're trying to do. I guess the the command-line tools eventually shouldn't be needed, so I'm ok with the default mode opening an empty editor.

You added the "Import map" option so that allowing an empty editor makes sense, but I think that should be converted to an "Open map" option, since importing/merging will be confusing while it's unfinished. For instance, what happens when I copy from one map and paste in the other? Which map is saved when I click save? There's a lot to it and it deserves its own PR I think.

Reply:
> The problem is this program is both a command-line tool and a windows app. If no arguments are given, should you see the 3D view or a help menu? Depends on what you're trying to do. I guess the the command-line tools eventually shouldn't be needed, so I'm ok with the default mode opening an empty editor.
> 
> You added the "Import map" option so that allowing an empty editor makes sense, but I think that should be converted to an "Open map" option, since importing/merging will be confusing while it's unfinished. For instance, what happens when I copy from one map and paste in the other? Which map is saved when I click save? There's a lot to it and it deserves its own PR I think.

Yes, first imported map = open. I don't know how to open bsp guy program without any map, just run render without add map ?

Reply:
It should work ok without any map loaded, or at least be simpler to add logic for.

I was thinking the "Import" window could just be renamed to "Open", and it would replace the current map instead of adding a new one. At least, I think that would be better until map merging in the editor is finished.